### PR TITLE
PBM-850 Documented cleanup of backup storage

### PR DESCRIPTION
**Pull request by nastena1606**
docs/reference/pbm-commands.md
	modified:   docs/usage/delete-backup.md
	modified:   docs/usage/schedule-backup.md

### CODE REVIEW

File: docs/usage/delete-backup.md [hunk 1]
```diff
@@ -1,6 +1,90 @@
...
+
+       ```{.bash .no-copy}
+       Snapshots:
+           2023-04-13T13:26:58Z 147.29MB <logical> [restore_to_time: 2023-04-13T13:27:15Z]
+           2023-04-13T10:12:08Z 147.29MB <logical> [restore_to_time: 2023-04-13T10:12:27Z]
+         PITR chunks [157.94KB]:
+           2023-04-13T10:12:28Z - 2023-04-13T13:27:46Z
+       ```
+
+    * When point-in-time recovery is enabled and you specify the timestamp greater than the `restore_to_time` for the most recent backup, PBM keeps this backup and all oplog slices deriving from it to ensure point-in-time recovery.
```
Comment (sandraromanchenko) [inline docs/usage/delete-backup.md:87]:
```suggestion
    * When point-in-time recovery is enabled and you specify the timestamp greater than the `restore_to_time` for the most recent logical backup, PBM keeps this backup and all oplog slices deriving from it to ensure point-in-time recovery. 
```

Comment (nastena1606) [reply docs/usage/delete-backup.md]: Updated

File: docs/usage/delete-backup.md [hunk 2]
```diff
@@ -1,6 +1,90 @@
...
+```
+
+### Behavior
+
+The timestamp you specify is considered to be the time to which you would wish to restore. Therefore, PBM doesn’t delete all backup snapshots and oplog slices that could be used to restore to this time.
+
+Here's how the cleanup works:
+
+* **Physical and selective backups** are deleted up to the specified time.
+* **Incremental physical backups** are deleted up to the specified time if the timestamp doesn't fall within the backup chain. If it does, PBM keeps the most recent base incremental backup and the whole chain deriving from it to ensure the potential restore.
```
[the remark this answers is not in the document]
Comment (nastena1606) [reply docs/usage/delete-backup.md]: Updated

**Review by sandraromanchenko — APPROVED** [review]

File: docs/reference/pbm-commands.md [hunk 3]
```diff
@@ -463,6 +463,23 @@ The command accepts the following flags:
...
+pbm delete-pitr [<flags>]
+```
+
+The command accepts the following flags:
+
+| Flag                     | Description               |
+| ------------------------ | ------------------------- |
+| `--older-than=TIMESTAMP` | Deletes backups older than date / time specified in the format:<br> - `%Y-%M-%DT%H:%M:%S` (e.g. 2020-04-20T13:13:20), <br> - `%Y-%M-%D` (e.g. 2020-04-20), <br> - `XXd` (e.g. 30d). Only days are supported|
+| `-w`, `--wait`           | Wait for the cleanup to finish. The flag blocks the shell session|
+| `-y`, `--yes`            | Cleans up the data storage without asking for a user's confirmation|
```
Comment (defbin) [inline docs/reference/pbm-commands.md:483]: `--dry-run` - "Report but do not delete"

**Review by defbin — CHANGES_REQUESTED** [review]

**Comment by defbin** [comment]
Also missed case: if there is a full logical, physical, or incremental backup at the `older-than` time, PBM does not exclude from deletion the most recent (before `older-than`) full logical backup and its oplog. The incremental chain is still excluded if needed.
The reason is that it is possible to fully restore data to the `older-than` time using any full backup of any type

**Review by sandraromanchenko — APPROVED** [review]

**Pull request merged**